CONTRIBUTING.md

```diff
@@ -10,20 +23,34 @@
 
 4. The tool/service has capabilities to analyse and improve software quality throughout the research software lifecycle, from development to long-term sustainability.
 
+### How to add or update a tool or service in the TechRadar
 
-## How to contribute 
+1. **Fork the repository**: Create a fork of the TechRadar repository on GitHub.
+2.  **Create a new branch**: Create a new branch for your changes.
+3. **Add or update JSON files**: Add new JSON files for new tools and services or update existing ones in the `data/software-tools` directory. Each JSON file should follow the structure defined in the [RS metadata schema](https://github.com/EVERSE-ResearchSoftware/schemas/tree/main/software). Please be as accurate and exhaustive as possible when filling in the metadata fields.
+4. **Commit your changes**: Commit your changes with a clear and descriptive commit message.
+5. **Push your changes**: Push your changes to your forked repository.
+6. **Create a pull request**: Open a pull request against the main branch of the TechRadar repository. Provide a clear explanation of the reasons to add or update the tool or service, and any relevant context or information that may help the curation team review your contribution.
+7. **Review and feedback**: The curation team will review your pull request. They may provide feedback or request changes before merging it into the main branch.
+8. **Merge and publish**: Once your pull request is approved, it will be merged into the main branch, and the changes will be reflected in the TechRadar dashboard at the next release.
```

Comment:
```suggestion
1. **Merge and publish**: Once your pull request is approved, it will be merged by the curation team into the main branch, and the changes will be reflected in the TechRadar dashboard at the next release.
```